package.json

```diff
@@ -18,6 +18,7 @@
     "url": "git@github.com:eea/volto-block-toc.git"
   },
   "dependencies": {
+    "github-slugger": "1.4.0",
```

Comment:
@tedw87  we shouldn't be installing this library, we have it in volto-anchors itself.